### PR TITLE
Update the Asp.Net core server generator to support Asp.net Core 2.1

### PR DESCRIPTION
**Pull request by SeanFarrow**
This PR updates the Asp.Net server generator to support Asp.Net core 2.1. It fixes issue #999.

### CODE REVIEW

**Comment by wing328** [comment]
## Upgrade Note

To fall back to ASP.NET core 2.0, please set `aspnetCoreVersion` to "2.0", e.g.
```json
{
  "aspnetCoreVersion": "2.0"
}
```
or
```bash
--additional-properties aspnetCoreVersion="2.0"
```

**Comment by cbornet** [comment]
I think the README must also be updated to 2.1

File: modules/openapi-generator/src/main/resources/aspnetcore/2.1/Dockerfile.mustache [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM microsoft/aspnetcore-build:2.0 AS build-env
```
Comment (wing328) [inline modules/openapi-generator/src/main/resources/aspnetcore/2.1/Dockerfile.mustache]: @SeanFarrow do we need to update the line 1 and 15 to use `microsoft/aspnetcore-build:2.1` instead?

**Comment by wing328** [comment]
@cbornet I believe you're referring to the project's README as I couldn't find "2.0" in `aspnetcore` auto-generated README, right?

**Comment by cbornet** [comment]
No, I refer to the generated README : https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/aspnetcore/2.1/README.mustache#L1

**Comment by SeanFarrow** [comment]
@wing328,
Ideally, yes.

**Comment by SeanFarrow** [comment]
@wing328,
I'll do this later today.

**Comment by wing328** [comment]
@SeanFarrow thanks again adding the ASP.net Core 2.1 support, which is included in the v3.3.0 minor release: https://twitter.com/oas_generator/status/1046941449609068544

**Pull request merged**